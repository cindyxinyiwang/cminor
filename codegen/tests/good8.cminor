/*
complext Expressions
*/

main: function integer ( argc: integer, argv: array [] string ) =
{
	i: boolean = true;
	j: boolean = false;
	k: boolean;
	
	k = i||j&&(!j)&&i&&(!j)||j;
	if (k == true)
		print "boolean arithmetic passed\n";
	
	return 0;
}
