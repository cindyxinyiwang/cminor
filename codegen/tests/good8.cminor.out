a resovles to param 0
